doc/userguide/upgrade/unified2.rst

```diff
@@ -0,0 +1,41 @@
+:orphan: Document not referenced in a toctree, so add this.
```

Comment:
Should we just add it to a toc somewhere?

Reply:
I tried but it never looked right..  I have another idea to try.